.gitignore

```diff
@@ -6,6 +6,7 @@
 # Folders
 _obj
 _test
+_manifests
```

Comment:
No, I often find it convenient to copy the example manifests to a separate directory to actually modify them for testing, however, I would like to avoid accidentally committing it :-)